packages/wonder-blocks-tooltip/components/tooltip-portal-mounter.js

```diff
@@ -1,158 +0,0 @@
-// @flow
```

Comment:
Very excited to see this go away!

Reply:
Also sad because of all the lessons and effort in getting it working.